src/components/Inputs/WalletPassphrase.tsx

```diff
@@ -64,9 +74,17 @@ const WalletPassphrase = ({ value, onChange, className }: Props) => {
         value={value}
         label={t`Optional passphrase`}
         type="password"
-        onChange={(e) => onChange(e.target.value)}
+        onChange={(e) => setValue(e.target.value)}
         disabled={!isConsentActive}
       />
+      <Input
+        value={confirmValue}
+        label={t`Confirm passphrase`}
+        type="password"
+        onChange={(e) => setConfirmValue(e.target.value)}
+        disabled={!isConsentActive || (!value && !confirmValue)}
```

Comment:
No need to check for `!confirmValue` here, no? In any case, if `!value`, we want to disable the confirm field IMO.

Reply:
I do need to check, because I don't want to end up at this state where I cannot delete the text:

<img width="390" alt="image" src="https://user-images.githubusercontent.com/1579899/183594762-893e5606-300f-4f06-aabc-acee1e91d7aa.png">

Reply:
done